src/scripting/components.ts

```diff
@@ -61,8 +61,17 @@ const buildBadge = (occupation) => {
   // SevenTV Stuff
   badgeContainer.setAttributeNode(document.createAttribute("data-v-9f956e7d"));
 
+  // Get the occupation object
+  const occupationObject = occupations.find((o) => o.apiValue === occupation);
```

Comment:
We changed branches and now the `occupation` variable passed in the scope returns the `SettingsOption` object:

```ts
export type SettingsOption = {
  name: string;
  slug: string;
  translation_key: string;
};
```